src/handleAction.js

```diff
@@ -1,19 +1,18 @@
 import isFunction from 'lodash/isFunction';
 import identity from 'lodash/identity';
 import isNil from 'lodash/isNil';
-import isSymbol from 'lodash/isSymbol';
+import includes from 'lodash/includes';
+import { ACTION_TYPE_DELIMITER } from './combineActions';
 
-export default function handleAction(type, reducers, defaultState) {
-  const typeValue = isSymbol(type)
-    ? type
-    : type.toString();
+export default function handleAction(actionType, reducers, defaultState) {
+  const actionTypes = actionType.toString().split(ACTION_TYPE_DELIMITER);
```

Comment:
Here is another solution:

We could leave this line like it was before and change only L14 to: 

```js
if (!actionType.include(action.type) {
```

We make use of [`String.prototype.includes()`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/String/includes) and we have to change less. Performance wise it would be less overhead imo.

Btw, why I `.toString()` needed when calling `action.type`?